test/mpi/matrix.cpp

```diff
@@ -207,25 +207,25 @@ TYPED_TEST(MatrixCreation, BuildFromExistingData)
     using Partition = typename TestFixture::Partition;
     using local_index_type = typename TestFixture::local_index_type;
     using matrix_data = gko::matrix_data<value_type, local_index_type>;
+    using input_triple =
+        gko::detail::input_triple<value_type, local_index_type>;
     using dist_mtx_type = typename TestFixture::dist_mtx_type;
     using dist_vec_type = gko::experimental::distributed::Vector<value_type>;
     using comm_index_type = gko::experimental::distributed::comm_index_type;
     auto rank = this->comm.rank();
     I<I<value_type>> res_local[] = {{{2, 0}, {0, 0}}, {{0, 5}, {0, 0}}, {{0}}};
     std::array<gko::dim<2>, 3> size_local{{{2, 2}, {2, 2}, {1, 1}}};
     std::array<matrix_data, 3> dist_input_local{
-        {{size_local[0], {{0, 0, 2}}},
-         {size_local[1], {{0, 1, 5}}},
-         {size_local[2],
-          std::initializer_list<
-              gko::detail::input_triple<value_type, local_index_type>>{}}}};
+        {{size_local[0], I<input_triple>{{0, 0, 2}}},
+         {size_local[1], I<input_triple>{{0, 1, 5}}},
+         {size_local[2]}}};
```

Comment:
```suggestion
         {size_local[2], I<input_triple>{}}}};
```
does it work?

Reply:
Yes, `matrix_data` has a constructor just with a size. In that case it has no non-zeros.